4.7.1-windowsservercore-1709/build/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# escape=`
+
+FROM microsoft/dotnet-framework:4.7.1-windowsservercore-1709
+
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+# Install Git
+RUN Invoke-WebRequest -UseBasicParsing https://github.com/git-for-windows/git/releases/download/v2.14.3.windows.1/MinGit-2.14.3-64-bit.zip -outfile git.zip; `
+    Expand-Archive git.zip -DestinationPath $Env:ProgramFiles\git; `
+    Remove-Item -Force git.zip; `
+    setx /M PATH $($Env:PATH + ';' + $Env:ProgramFiles + '\git')
+
+# Install NuGet CLI
+RUN New-Item -Type Directory $Env:ProgramFiles\NuGet; `
+    Invoke-WebRequest -UseBasicParsing https://dist.nuget.org/win-x86-commandline/v4.3.0/nuget.exe -OutFile $Env:ProgramFiles\NuGet\nuget.exe; `
+    setx /M PATH $($Env:PATH + ';' + $Env:ProgramFiles + '\NuGet')
+
+# Install Targeting Pack
+RUN Invoke-WebRequest https://download.microsoft.com/download/9/0/1/901B684B-659E-4CBD-BEC8-B3F06967C2E7/NDP471-DevPack-ENU.exe -OutFile NDP462-DevPack-KB3151934-ENU.exe; `
```

Comment:
Not that is matters functionally but you should keep consistent file names after download.  Here you are downloading NDP471-DevPack-ENU.exe and calling it NDP462-DevPack-KB3151934-ENU.exe

Reply:
This wasn't intentional - thanks for pointing it out - fixed.